app/src/androidTest/java/com/github/studydistractor/sdp/user/FakeCreateUserModule.kt

```diff
@@ -0,0 +1,40 @@
+package com.github.studydistractor.sdp.user
+
+import androidx.compose.runtime.snapshots.SnapshotStateList
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.android.components.ActivityComponent
+import dagger.hilt.testing.TestInstallIn
+import javax.inject.Inject
+
+@Module
+@TestInstallIn(
+    components = [ActivityComponent::class],
+    replaces = [UserModule ::class]
+)
+
+object FakeCreateUserModule {
+
+    @Provides
+    fun provideFakeCreateUserModule() : UserService{
+        return FakeCreateUserImplementation()
+    }
+}
+
```

Comment:
You don't need a Hilt module if you are never doing injection. I think we may be moving away from Hilt when everything will be screens since we can pass the interface around now.